lib/swarm/distribution/static_quorum_ring.ex

```diff
@@ -0,0 +1,76 @@
+defmodule Swarm.Distribution.StaticQuorumRing do
+  @moduledoc """
+  A quorum is the minimum number of nodes that a distributed cluster has to obtain in order to be
+  allowed to perform an operation. This can be used to enforce consistent operation in a distributed system.
+
+  ## Quorum size
+
+  You must configure the quorum size using the `:static_quorum_size` setting:
+
+      config :swarm,
```

Comment:
May be worth including the `:distribution_strategy` option here for clarity and in order to re-emphasize that both options are required to use this strategy correctly.